server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala

```diff
@@ -31,6 +23,9 @@ trait AkkaHttpServerInterpreter {
 
   implicit def executionContext: ExecutionContext
 
+  protected def requestBody: (Materializer, ExecutionContext) => RequestBody[Future, AkkaStreams] = new AkkaRequestBody(akkaHttpServerOptions)(_, _)
```

Comment:
isn't extracting these two methods adding more code than just duplicating the `new ...` invocation? :)

Reply:
ah you want to override those ... maybe extracting a protected `toRoute` method would be better? then both this and grpc servers could delegate there